File: Implementare Cursuri Angular/Curs 4/Inventory/src/app/app-logic/inventory-item.ts
export class InventoryItem {
    id!: number;
    name!: string;
    user!: string;
    description!: string;
    location!: string;
    inventoryNumber!: number;
    createdAt!: Date;
    modifiedAt!: Date;
    deleted!: boolean;

    constructor(item?: Partial<InventoryItem>) {
        if (item) {
            Object.assign(this, item);
        }
    }
}

File: Curs frontend - backend/Backend - InventoryApp/InventoryApp/Models/InventoryItem.cs
namespace InventoryApp.Models
{
    public class InventoryItem
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string User {  get; set; }

        public string Location {  get; set; }

        public int InventoryNumber { get; set; }

        public DateTime CreatedAt {  get; set; }

        public DateTime ModifiedAt {  get; set; }

        public bool Deleted { get; set; }

    }
}
